Justfile

```diff
@@ -21,7 +21,7 @@ clean:
   rm -rf site
 
 # Serve docs site (unversioned, watches for changes)
-serve: (_run "mkdocs serve -a 0.0.0.0:8000")
+serve: (_run "mkdocs serve --strict -a 0.0.0.0:8000")
```

Comment:
I'm all in for the strict mode, see https://github.com/githedgehog/docs/pull/95, but this is gonna break until we fix the links that are currently broken - see https://github.com/githedgehog/fabric/issues/809.

Until then, how about passing optional arguments to the `serve` target, as proposed for the `build` target in https://github.com/githedgehog/docs/pull/95/files#diff-2f90408c2b0302b1cdb7f5d634114750837f940fa82d13057d9c18d0170a7e5cR17?